src/datatype.rs

```diff
@@ -21,6 +22,67 @@ const EXCEL_1900_1904_DIFF: f64 = 1462.;
 #[cfg(feature = "dates")]
 const MS_MULTIPLIER: f64 = 24f64 * 60f64 * 60f64 * 1e+3f64;
 
+/// A struct that combines cell value and style information
+#[derive(Debug, Clone, PartialEq, Default)]
+pub struct CellData {
+    /// The cell value
+    pub value: Data,
+    /// The cell style
+    pub style: Option<Style>,
```

Comment:
Strictly speaking the style is never optional in the Excel file formats. There is always an implicit 0 style that is the default and is defined in the `theme.xml` file. Note, I'm not necessarily saying that it shouldn't be an `Option<>`.